.husky/pre-commit

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-# `tsc` so our ESLint rules are compiled to `./dist` before the linter runs.
-npx tsc -outDir ./dist && pnpm lint-staged
+# `typecheck` so our ESLint rules are compiled to `./dist` before the linter runs.
+NODE_ENV=production pnpm typecheck
```

Comment:
Same behavior. The `&&` wasn't adding anything and `pnpm typecheck` can be used instead.